content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/README.md

```diff
@@ -0,0 +1,59 @@
+<!--
+Copyright 2019 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Container (v1)
+====
+Container component written in HTL.
+
+## Features
+
+* Allows addition of Background Color and Background image of varying container.
```

Comment:
I'd reformulate:
Allows to configure a background color and a background image.

Reply:
review comments incorporated